pontoon/base/tests/models/test_repository.py

```diff
@@ -2,26 +2,10 @@
 from unittest.mock import call, patch, Mock
 
 import pytest
-from urllib.parse import urlparse
 
 from pontoon.test.factories import ProjectLocaleFactory
 
 
-@pytest.mark.django_db
-def test_repo_checkout_path(repo_file, settings):
-    """checkout_path should be determined by the repo URL."""
-    # im a bit unclear about the mix of os.path and urlparse here
-    # how would this work on windows <> linux ?
-    assert repo_file.checkout_path == os.path.join(
-        *[repo_file.project.checkout_path] + urlparse(repo_file.url).path.split("/")
-    )
-    settings.MEDIA_ROOT = "/media/root"
-    assert repo_file.checkout_path == os.path.join(
-        *[repo_file.project.checkout_path] + urlparse(repo_file.url).path.split("/")
-    )
-    assert repo_file.project.checkout_path.startswith("/media/root")
-
-
```

Comment:
I'm curious why did you choose to remove this test case.

Reply:
It was in error - when skimming the test, I thought it did something `file`-specific. I have re-added the test (now using `repo_git`).